InterGraph/interaction_graph.py

```diff
@@ -0,0 +1,404 @@
+import os.path
+import pandas as pd
+import os
+import pickle
+import urllib
+import pandas as pd
+import numpy as np
+import networkx as nx
+import openbabel
+
+from tqdm import tqdm
+from rdkit import Chem
+from scipy.spatial.distance import cdist
+from itertools import product
+from openbabel import openbabel
+
+pdb_raw_d = "../data/" + "pdb_raw"
+
+
+def file_path():
+    global pdb_raw_d
```

Comment:
I am not super sure why you call global here --- can you give some context on this decision?